crates/fluvio-cluster/src/check/mod.rs

```diff
@@ -233,6 +235,15 @@ pub enum UnrecoverableCheckStatus {
     #[error("Local Fluvio cluster wasn't deleted. Use 'resume' to resume created cluster or 'delete' before starting a new one")]
     CreateLocalConfigError,
 
+    /// The installed version of the local cluster is incompatible
+    #[error("Must have local platform version {required}. You have {installed}.")]
```

Comment:
Right now it looks like required and version are swapped.

```
$ fluvio version
 Fluvio CLI                     : 0.11.9-dev                                                       
 Fluvio CLI Arch                : x86_64-unknown-linux-gnu                                         
 Fluvio CLI SHA256              : b2173e8f9c5a20b591642063dec21f8dc44f536e3758ab62d0581f050a8eae11 
 Fluvio Channel Frontend SHA256 : b2173e8f9c5a20b591642063dec21f8dc44f536e3758ab62d0581f050a8eae11 
 Fluvio Platform                : Not available (local)                                            
 Git Commit                     : 91b3a472eecfa4e85951701743f0dd6840c42159                         
 OS Details                     : Pop!_OS 22.04 (kernel 6.8.0-76060800daily20240311-generic) 
```
But on resume I get:

```
$ fluvio cluster resume
     ✅ Local Fluvio is not installed
    ❌ Check Versions match failed Must have local platform version 0.11.9-dev. You have 0.11.8.
```
It's also a little confusing wording:  Does this sound better? "Check Versions match failed: cannot resume a 0.11.8 cluster with fluvio version 0.11.9-dev."

Honestly, "Local fluvio not installed" is a little confusing too, but that wasn't touched with this PR so it's a separate concern.